insights/parsers/ls_var_lib_nova_instances.py

```diff
@@ -48,7 +48,8 @@
     6
     >>> ls_var_lib_nova_instances.dir_entry('/var/lib/nova/instances/locks', 'nova-storage-registry-lock')['raw_entry']
     '-rw-r--r--. nova nova system_u:object_r:nova_var_lib_t:s0 nova-storage-registry-lock'
-
+    >>> ls_r_var_lib_nova_instances.dir_entry('/var/lib/nova/instances/e560e649-41fd-46a2-a3d2-5f4750ba2bb4', 'console.log')['size']
```

Comment:
@psachin - Would you please update the docstring? since this python file includes multiple parsers, just lik https://github.com/RedHatInsights/insights-core/blob/master/insights/parsers/systemctl_show.py
- update the title
- move the `Examples` to each Parser class correspondingly

Thank you.